docs/yaml_docs/spatial_qc.md

```diff
@@ -28,7 +28,7 @@ For more information on functionalities implemented in `panpipes` to read the co
 
 
 <span class="parameter">resources</span><br>
-Computing resources to use, specifically the number of threads used for parallel jobs.
+Computing resources to use, specifically the number of threads used for parallel jobs.  check [threads_tasks_panpipes.csv](https://github.com/DendrouLab/panpipes/blob/threads_doc_g/docs/yaml_docs/threads_tasks_panpipes.csv) for more information on which threads each specific task requires.
```

Comment:
@giuliaelgarcia you deleted the csv file in favour of the md file but you're still referencing the csv in all the documentation pages. can you please fix?

Reply:
@bio-la Just fixed this!